src/modules/forms/FormInput/FormInput.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+import FormField from '../FormField';
+import FormLabel from '../FormLabel';
+
+import './FormInput.css';
+
+const propTypes = {
+  attachTo: PropTypes.string.isRequired,
+  component: PropTypes.string.isRequired,
+  disabled: PropTypes.bool,
+  label: PropTypes.string.isRequired,
+  type: PropTypes.string.isRequired,
+  normalize: PropTypes.func,
+  tooltip: PropTypes.string,
+};
+
+const renderField = (component, type, name, normalize, parse, disabled) => {
+  const props = {
+    name,
+    component,
+    type,
+    normalize,
+    parse,
+    disabled,
+  };
+  return (
+    <Field className="j-input" {...props} />
+  );
+};
+
+const FormInput = props => {
+  const { attachTo, component, type, tooltip, normalize, parse, disabled } = props;
+
+  return (
+    <FormField>
+      <FormLabel htmlFor={attachTo} text={props.label} tooltip={tooltip} />
+      {renderField(component, type, attachTo, normalize, parse, disabled)}
```

Comment:
or maybe just use Field component?
can't get the point of having additional function